spec/swagger.yaml

```diff
@@ -101,6 +101,28 @@ tags:
 
       The Jobs API allows you to create a job, add a file to a job, authorize a job, and cancel a job, among many other actions. You have full control via the API to manage your job(s).
 
+  - name: "Jobs Batches"
```

Comment:
I think it should be Job Batches. My 2 cents curious what heidi thinks

Reply:
Agree with Greg. Should be "Job Batches".

Reply:
done